### PR TITLE
fix(curriculum): typo in the description in Problem 29: Distinct powers

### LINKED ISSUE #44242: Mistake in the description of Problem 29: Distinct powers
## Affected page

Coding Interview Prep > Project Euler > Problem 29: Distinct powers
https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-29-distinct-powers

## Expected behavior

The description says `ab` but I think it should be `a^b`

## Screenshots

project_euler_29_issue

## System

- Device: Laptop
- OS: Windows 10
- Browser: chrome
- Version: 96.0.4664.45

## Additional context

Originally reported in a comment on Crowdin duing translation.
https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/21122/en-ja/132?filter=basic&value=3#q=623850

### PR DESCRIPTION
**Pull request by MxDui**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44242

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dashedName: problem-29-distinct-powers
 
 # --description--
 
-Consider all integer combinations of ab for 2 ≤ a ≤ 5 and 2 ≤ b ≤ 5:
+Consider all integer combinations of a^b for 2 ≤ a ≤ 5 and 2 ≤ b ≤ 5:
```
Comment (gikf) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md:11]:
```suggestion
Consider all integer combinations of $a^b$ for 2 ≤ a ≤ 5 and 2 ≤ b ≤ 5:
```

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ If they are then placed in numerical order, with any repeats removed, we get the
   4, 8, 9, 16, 25, 27, 32, 64, 81, 125, 243, 256, 625, 1024, 3125
 </div>
 
-How many distinct terms are in the sequence generated by `ab` for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
+How many distinct terms are in the sequence generated by `a^b` for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
```
Comment (gikf) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md:26]:
```suggestion
How many distinct terms are in the sequence generated by $a^b$ for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
```

**Review by gikf — COMMENTED** [review]
I've added suggestions to make it look nicer by using MathJax.

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ If they are then placed in numerical order, with any repeats removed, we get the
   4, 8, 9, 16, 25, 27, 32, 64, 81, 125, 243, 256, 625, 1024, 3125
 </div>
 
-How many distinct terms are in the sequence generated by `ab` for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
+How many distinct terms are in the sequence generated by `$a^b$` for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
```
Comment (gikf) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-29-distinct-powers.md:26]:
```suggestion
How many distinct terms are in the sequence generated by $a^b$ for 2 ≤ `a` ≤ `n` and 2 ≤ `b` ≤ `n`?
```

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Review by naomi-lgbt — APPROVED** [review]
Thanks for taking care of this!

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**